extensions/README.md

```diff
@@ -33,15 +35,38 @@ can be aware of it.
 
 Each extension has at least one *owner*. You can find extension owners in each extension's README.
 
-## General Conventions
-
-1. Additional attributes relating to an [Item](../item-spec/item-spec.md) should be added into the Item Properties object,
-   rather than directly in the Item object.
-2. In general, additional attributes that apply to an Item Asset should also be allowed in Item Properties and vice-versa.
-   For example, the `eo:bands` attribute may be used in Item Properties to describe the aggregation of all bands available in
-   the Item Asset objects contained in the Item, but may also be used in an individual Item Asset to describe only the bands available in that asset.
-3. Additional attributes relating to a [Catalog](../catalog-spec/catalog-spec.md) or
-   [Collection](../collection-spec/collection-spec.md) should be added to the root of the object.
+## Using Extensions
+
+When deciding how to model data in STAC it is highly recommended to first look at the [list of 
+extensions](https://stac-extensions.github.io/) and re-use fields there instead of creating your own version. This
+increases interoperability, as users know that the meaning of your fields is the same as in other STAC 
+implementations. Many clients will also understand more mature extensions for better display and querying. 
+
+To incorporate an extension in STAC the 'extension ID' of the extension must be added to the `stac_extensions`
+array of the STAC [Catalog](../catalog-spec/catalog-spec.md#stac_extensions), 
+[Collection](../collection-spec/collection-spec.md#stac_extensions) or [Item](../item-spec/item-spec.md#stac_extensions)
+object. This identifier is a link to the JSON Schema URL that validates the fields in the extension, so STAC validators
+can fetch the Schema to validate that the STAC object properly follows the extension. These JSON Schema URLs also act as 
+identifiers for specific version of the extension that the STAC object implements. The extension ID can be
```

Comment:
```suggestion
identifiers for specific version of the extension that the STAC object implements. The extension ID can usually be
```

Not all (3rd party) extensions may follow our template.

Reply:
Good point